src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -430,12 +430,12 @@ class Project: ADOSVTBase
                     $controlResult.AddMessage([VerificationResult]::Passed,"Number of administrators configured meet the minimum required administrators count: $($this.ControlSettings.Project.MinPAMembersPermissible)");
                  
                 }
-                if($TotalPAMembers -gt 0){
+                #if($TotalPAMembers -gt 0){
```

Comment:
Why this condition is removed?

Reply:
same already checked above and this line comes under that